source/includes/write/transaction.py

```diff
@@ -0,0 +1,35 @@
+# start-transaction
+# Establish a connection to the MongoDB server
```

Comment:
S: Based on the [code comment style guide](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit?usp=sharing) (lmk if you need access), I'd change the tense of these code comments
```suggestion
# Establishes a connection to the MongoDB server
```